testing-guidance/workflow-tests/o3de-project-manager/remote-templates/remote-templates-workflow-tests.md

```diff
@@ -2,9 +2,18 @@
 
 These workflows center around testing the basic functionality of the Project Manager remote template acquisition tools.
 
+## Documentation & Tutorials
+
+* [Workshop: Building a Networked Game from Scratch...](https://youtu.be/4f4olmUo44k)
+
 ## Prerequisites
 
-*   Locally installed tools for the remote source being used (git installed with credentials for GitHub, for example)
+* Locally installed tools for the remote source being used (git installed with credentials for GitHub, for example)
+* A GitHub repository that has a remote template.
```

Comment:
GitHub isn't required. 

Suggestion:
A Git repository that has a remote template.
The git repository must contain a repo.json at its root which points to templates.
The repo.json contains a template list which points to the project templates within said repository.
See https://github.com/o3de/o3de-extras/blob/development/repo.json as an example.
```
"templates": [
        "https://github.com/o3de/o3de-extras.git/Templates/Multiplayer",
        "https://github.com/o3de/o3de-extras.git/Templates/Ros2ProjectTemplate"
    ]
```
Each template folder must contain a template.json at its root which describes the template. Including information such as where the template zip can be downloaded, and the zip's SHA256 hash value for security
```
{
    "template_name": "Multiplayer",
    "origin": "Open 3D Engine - o3de.org",
    "origin_uri": "https://github.com/o3de/o3de-extras/releases/download/1.0/Template_Multiplayer-1.0.zip",
    "sha256": "c1d1f6e260c6121f013b4d54e213e0c7f0f99281c66ebe48bb6403a1180ee444",
```
See https://github.com/o3de/o3de-extras/blob/development/Templates/Multiplayer/template.json as example.